src/main/java/io/camunda/testing/assertions/MessageAssert.java

```diff
@@ -49,6 +50,28 @@ public MessageAssert hasBeenCorrelated() {
     return this;
   }
 
+  /**
+   * Verifies the expectation that a message start event has been correlated
+   *
+   * @return this {@link MessageAssert}
+   */
+  public MessageAssert hasMessageStartEventBeenCorrelated() {
```

Comment:
:x: this sounds wrong. The message start event is not correlated, the message is.

rename to: `hasCreatedProcessInstance` 
Also adjust error message accordingly